server/sonar-server/src/main/java/org/sonar/server/measure/ws/ComponentAction.java

```diff
@@ -172,12 +174,13 @@ private static ComponentWsResponse buildResponse(ComponentWsRequest request, Com
       MetricDto metric = metricsById.get(measure.getMetricId());
       measuresByMetric.put(metric, measure);
     }
-    Map<Long, ComponentDto> referenceComponentUuidById = new HashMap<>();
     if (refComponent.isPresent()) {
-      referenceComponentUuidById.put(refComponent.get().getId(), refComponent.get());
+      response.setComponent(componentDtoToWsComponent(component, measuresByMetric, singletonMap(refComponent.get().uuid(), refComponent.get())));
+    }
+    else {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this "else" on the same line that the previous closing curly brace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)